### ISSUE TITLE
Opening Raya Lucaria Academy causes a crash

### ISSUE THREAD

**Issue by Ascyius** [googleben/ERMapViewer#1]
Also occurs with several other maps.

Is there any plan to allow exporting?

**Comment by googleben** [comment]
I believe all crashes are fixed in the current code, but I know for sure the fixes aren't in the current release; it's mainly down to not parsing the .msb files correctly IIRC. As for exporting, it might happen in the future but for now you could hack something together with my fork of SoulsFormats.

**Comment by JaseWorthing** [comment]
Do you have any plans to update with a new release? I attempted to make my own build with the current code, but ran into issues.

**Comment by googleben** [comment]
Not really, I'd recommend you use [DSMapStudio](https://github.com/soulsmods/DSMapStudio) @JaseWorthing

**Comment by JaseWorthing** [comment]
Gotcha. Thanks for the prompt reply!
